### ISSUE TITLE
When deleting a "paragraph", it reappears below at the next `<br />`

### ISSUE THREAD

**Issue by Pitouli** [samclarke/SCEditor#443]
In the xHTML mode, on Safari/Chrome but NOT on Firefox, I observe the following and extremely annoying bug.

Consider we have in the editor 2 paragraphs, and in the second one there is one or more breakline. 
> Paragraph1
>
> Paragraph2 first line
> Paragraph2 second line

If I want to have only one paragraph, I will naturally put my cursor at the beginning of line _Paragraph2 first line_, and press the delete key. Therefore, or the `</p><p>` should be replace by a `<br />` and I will obtain:
> Paragraph1
> Paragraph2 first line
> Paragraph2 second line

Or it should be simply deleted and I will obtain: 
> Paragraph1 Paragraph2 first line
> Paragraph2 second line

Depending of the case and of what I want in the end, I will then delete/add/do-nothing with the break line...

The issue is, on Safari/Chrome, when I do what I described, I obtain this:
> Paragraph1
> Paragraph2 first line
>
> Paragraph2 second line

That means the `</p><p>` is not deleted but moved one line below... 
Basically, that means that *you cannot delete a paragraph* which is something quit annoying for this kind of tool.

To be completely precise, you can delete the paragraph if you make it "go down" until it merges with another paragraph. That means that if the paragraph you want to merge has 6 `<br />` you need to do six times "place the cursor, delete the paragraph, add a breakline _(because on Safari/Chrome the default behavior does not replace the paragraph with a breakline)_"...

Any idea how to fix this?
